### PR TITLE
style: homepage ui change

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by Riddhiagrawal001 — DISMISSED** [review]

**Review by PritishBudhiraja — DISMISSED** [review]

**Review by PritishBudhiraja — DISMISSED** [review]

**Review by PritishBudhiraja — DISMISSED** [review]

File: src/screens/HyperSwitch/Home/HomeV2.res [hunk 1]
```diff
@@ -397,9 +397,21 @@ module QuickStartModule = {
...
+  let {isProdIntentCompleted} = React.useContext(GlobalProvider.defaultContext)
+  let enumDetails = Recoil.useRecoilValueFromAtom(HyperswitchAtom.enumVariantAtom)
+  let typedEnumValue = enumDetails->LogicUtils.safeParse->QuickStartUtils.getTypedValueFromDict
   <div className="w-full flex flex-col gap-14">
     <QuickStartModule />
-    <RecipesAndPlugins />
+    <UIUtils.RenderIf
+      condition={!(
+        [
+          typedEnumValue.testPayment.payment_id->LogicUtils.isNonEmptyString,
```
Comment (Riddhiagrawal001) [inline src/screens/HyperSwitch/Home/HomeV2.res:408]: Can we assign all the condition in one variable and replace the variable here for better readability

Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Home/HomeV2.res]: Updated

**Review by PritishBudhiraja — APPROVED** [review]

**Review by Riddhiagrawal001 — APPROVED** [review]

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by vsrivatsa-juspay — DISMISSED** [review]

**Review by PritishBudhiraja — DISMISSED** [review]

File: src/screens/HyperSwitch/Home/HomeV2.res [hunk 2]
```diff
@@ -397,15 +397,21 @@ module QuickStartModule = {
...
-      <QuickStartModule />
+  let {isProdIntentCompleted} = React.useContext(GlobalProvider.defaultContext)
+  let enumDetails = Recoil.useRecoilValueFromAtom(HyperswitchAtom.enumVariantAtom)
+  let typedEnumValue = enumDetails->LogicUtils.safeParse->QuickStartUtils.getTypedValueFromDict
+  let showRecipesAndPlugins =
+    [
+      typedEnumValue.testPayment.payment_id->LogicUtils.isNonEmptyString,
+      typedEnumValue.integrationCompleted,
+      isProdIntentCompleted->Option.getOr(false),
+    ]->Array.includes(false)
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Home/HomeV2.res:408]: there will be UI if you directly check for these values.Please refer to QuickStartModule

Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Home/HomeV2.res]: updated

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**